### PR TITLE
DEV-1756 Modify/clean up toggle card

### PR DESCRIPTION
**Pull request by SKona306**
This PR will:

- Add binding to allow user to control whether icon is visible on toggle card
- Modify loading spinner to replace toggle while loading

### Wait to merge until:

- Design review with Alex
- Other PR in UI-shared-components is merged
- version is updated

### CODE REVIEW

File: app/components/app-toggle-card.component.scss [hunk 1]
```diff
@@ -86,13 +94,13 @@
 // color options for toggle card
 
 .toggle-success-green {
-    color: green;
-    border-left-color: green;
+    color: green !important;
+    border-left-color: green !important;
```
Comment (Morkos) [inline app/components/app-toggle-card.component.scss]: where is the conflict that makes have to put !important?

File: app/components/toggle-card/app-toggle-card.component.html [hunk 2]
```diff
@@ -45,7 +46,7 @@
                     </div>
                 </div>
             </div>
-            <div>
+            <div ng-if="vm.isIconShowing">
```
Comment (Morkos) [inline app/components/toggle-card/app-toggle-card.component.html]: isIconVisible

File: app/components/toggle-card/app-toggle-card.component.ts [hunk 3]
```diff
@@ -2,14 +2,15 @@ import angular from 'angular';
...
-    iconClass: string;
+    iconClass?: string;
     cardTitle: string;
     tooltipPlacement?: string;
     tooltipMessage?: string;
     message?: string;
     isEnabled: boolean;
     onSwitchClick: () => null;
     isToggleLoading: boolean;
+    isIconShowing: boolean;
```
Comment (Morkos) [inline app/components/toggle-card/app-toggle-card.component.ts]: isIconVisible

**Review by Morkos — CHANGES_REQUESTED** [review]
Few comments below

File: app/components/app-toggle-card.component.scss (hunk 1 above)
Comment (SKona306) [reply app/components/app-toggle-card.component.scss]: removed. not needed after pulling changes from main branch

**Review by JStein92 — APPROVED** [review]
Nice work!

**Pull request merged**